software/spmd/bypass_core/Makefile

```diff
@@ -2,17 +2,6 @@
 bsg_tiles_X = 4
 bsg_tiles_Y = 4
 
-#########################################################
-#each bytes defines the core type
-# 00: vanilla core
-# 01  vscale  core
-TYPE_VEC="64'h00_00_00_00"
```

Comment:
Is type_vec still a thing we support?

Reply:
nope

Reply:
This is related to the machine config. It should be supported in machine.include.

Reply:
OK, so we have that in other makefiles too, right?